profile/README.md

```diff
@@ -0,0 +1,19 @@
+## Welcome to thoughtbot on GitHub
+
+[thoughtbot] is known worldwide as a leading [design], [development], and
+[product management] consultancy providing end-to-end services along the
+complete product development life cycle. thoughtbot validates, builds, scales,
+maintains, and supports web and mobile products.
```

Comment:
Some suggesttions to shorten the intro paragraph and expand in the second paragraph. I like the links you have included - keep those!

"thoughtbot is your expert partner for product design and development. We bring digital products from idea to success and teach you how because we care."

Reply:
I spoke too soon about the links - we don't have a development overview page that can be neatly linked to for "development". Perhaps we link "product design and development" to https://thoughtbot.com/services.